docs/worker-node/install-wn-oasis.md

```diff
@@ -5,7 +5,7 @@ Installing the Worker Node Client via OASIS
 ===========================================
 
 The **OSG Worker Node Client** is a collection of software components that is expected to be added to every worker node
-that can run OSG jobs. It provides a common environment and a minimal set of common tools that all OSG jobs can expect
+that can run OSPool jobs. It provides a common environment and a minimal set of common tools that all OSPool jobs can expect
```

Comment:
I think OSG jobs is actually ok here. It's not just OSPool, the WN client is a combination of requirements from all the collabs

Reply:
OK, if "OSG jobs" is allowed then I'll keep it.